src/components/Badge/Badge.jsx

```diff
@@ -16,7 +17,7 @@ export const Badge = ({
     {...transferProps(restProps)}
     className={classNames(
       styles.root,
-      priority === 'outline' && styles.isRootPriorityOutline,
+      getRootPriorityClassName(priority, styles),
```

Comment:
CSS classes for priority=filled are missing in Badge.module.scss.

Reply:
I got confused as @atmelmicro. While CSS classes are missing in this pull request, their are added in following pull request: https://github.com/react-ui-org/react-ui/pull/562

@adamkudrna, if you can, leave the information about it somewhere (probably in description). To be honest, it should be one PR as this leave unfinished feature in the code.

Closing comment and approving PR.

Reply:
Of course you are right, sorry, I didn't notice it. I tried to split my work into as many PRs as possible and this thing leaked 🙂.

BTW, moments like this make me feel we'd appreciate visual tests here in RUI.

Reply:
**UPDATE:** OK, I checked the rendered output and guess what, it looks correct 🎉. So what's happening here is that calling of a non-existing class name in CSS modules is not reported. When I think of it, I should have added the priority class check to tests and then I would know. 🤔 

<img width="847" alt="obrazek" src="https://github.com/user-attachments/assets/5e3aca7f-8b6a-4279-884f-1c0c538ad8aa">

Anyway, thanks to both of you for catching this. I'll fix it and add a test (and then probably use my admin power to merge it so the PR cascade gets smaller).

Reply:
Didn't find anything, maybe the classes are just resolved differently in the test environment (which sounds terribly), I don't know 🤷🏻‍♂️.

Anyway, I added the missing class to CSS.

Reply:
If tests does not fail when they should, consider creating issue to look at it.